ENIGMAsystem/SHELL/Graphics_Systems/OpenGLES/d3d.cpp

```diff
@@ -12,7 +12,9 @@ void graphics_state_flush_samplers() {
     glActiveTexture(GL_TEXTURE0 + i);
     glBindTexture(GL_TEXTURE_2D, tex);
     if (tex == 0) continue; // texture doesn't exist skip updating the sampler
+    #ifdef GL_TEXTURE_WRAP_R
     glTexParameteri(GL_TEXTURE_2D, GL_TEXTURE_WRAP_R, GL_CLAMP_TO_EDGE);
+    #endif
```

Comment:
If you like, R can simply be deleted for now. It is only supported in GLES 3 and it has to do with cube/3D textures which ENIGMA and GM do not handle.

Reply:
deleted it